src/validation.cpp

```diff
@@ -4258,14 +4258,15 @@ bool CVerifyDB::VerifyDB(const CChainParams& chainparams, CCoinsView *coinsview,
     int nGoodTransactions = 0;
     BlockValidationState state;
     int reportDone = 0;
+    const int log_step{nCheckDepth > 1000 ? 1 : 10};
```

Comment:
Perhaps place the explanatory comment of `// report every 10% step if nCheckDepth is <= 1000 blocks else report every 1% step` here where you initialize `log_step`.